python/paddle/trainer_config_helpers/layers.py

```diff
@@ -235,16 +235,16 @@ class AggregateLevel(object):
 
     Accordingly, AggregateLevel supports two modes:
 
-    - :code:`AggregateLevel.EACH_TIMESTEP` means the aggregation acts on each
+    - :code:`AggregateLevel.TO_NO_SEQUENCE` means the aggregation acts on each
```

Comment:
NO_SEQUENCE => NON_SEQUENCE
We can still keep the original names for compatibility.